commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -148,9 +140,21 @@ def questions(self) -> Questions:
                 ),
             },
         ]
-        return questions
+
+        # TODO: How would filter functions would be handled from config?
+        return self.config.settings.get("questions", questions)
```

Comment:
would like to consult on this

Reply:
yes, what was it?

Reply:
Since questions can have filter functions, how would they be handled read from a config file? From my understanding, this was also not handled in the past

Reply:
yes, this was not handled. IMO, if we're to build a complicate customization, we should fallback to implement a CZ class. Using config file is just a handy way to achieve easy ones